content/en/Platform Deep Dive/Credits/ptaas-tiers.md

```diff
@@ -12,7 +12,7 @@ We offer three [PtaaS tiers](https://www.cobalt.io/pentest-pricing) to best suit
 
 To get started, check our [pricing model](https://www.cobalt.io/pentest-pricing), and select a plan that matches your testing expectations and business needs.
 
-| Standard | Premium | Enterprise |
+| <span style="border-radius:10px;-webkit-border-radius:10px;-moz-border-radius:10px;background-color:#4185EB;padding:10px;text-align:center;opacity:75%;"><span style="color: #FFFFFF;font-weight:bold;">Standard</span></span> | <span style="border-radius:10px;-webkit-border-radius:10px;-moz-border-radius:10px;background-color:#8E3EDB;padding:10px;text-align:center;opacity:75%;"><span style="color: #FFFFFF;font-weight:bold;">Premium</span></span> | <span style="border-radius:10px;-webkit-border-radius:10px;-moz-border-radius:10px;background-color:#0DB285;padding:10px;text-align:center;opacity:75%;"><span style="color: #FFFFFF;font-weight:bold;">Enterprise</span></span> |
```

Comment:
Brainstorming... Is it possible to set up a shortcode for each label?
- Alternative: we'll need a place to "copy/paste" the HTML for each label, maybe even the confluence page for the UI Style Guide.

Maybe we should also chat with a designer? Post this idea in an internal Slack channel?

Reply:
@mjang-cobalt I added shortcodes for labels, they work.

> Alternative: we'll need a place to "copy/paste" the HTML for each label, maybe even the confluence page for the UI Style Guide.

I didn't understand why we need these labels in the UI Text Style Guide.

> Maybe we should also chat with a designer? Post this idea in an internal Slack channel?

How can designers help us? If they suggest a different look and feel, I may not be able to handle it technically.

Initially, I thought we don't need to add these labels to each page. We can only add them to the pages describing Premium or Enterprise features: Jira and GitHub integrations and customizable reports.